### DISCUSSION TITLE
gh-pages deployment trouble

### DISCUSSION THREAD

**Discussion by 2-GO** [community/community#62304] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello Community, 

I have encountered an error in deploying my e-commerce project on gh-pages. The deployment works fine on localhost, but upon deploying, the link shows a Nextjs prompt page. Any suggestions on I can fix this? "https://github.com/2-GO/e-commerce"

**Comment by AMohamedAakhil** [comment, marked as answer]
Github pages only supports static websites. You cannot host your next.js application here. Use vercel instead: https://vercel.com/new

**Reply by 2-GO** [reply]
I was not aware. Thanks @AMohamedAakhil
